src/main/java/io/reactivex/Single.java

```diff
@@ -344,6 +344,32 @@
         return Flowable.fromArray(sources).concatMapEager(SingleInternalHelper.<T>toFlowable());
     }
 
+    /**
+     * Concatenates a Publisher sequence of Publishers eagerly into a single stream of values.
```

Comment:
SingleSources

Reply:
Could you also fix the JavaDoc of the `Maybe` version?

Reply:
good idea, will do